RadixWallet/Clients/SecurityCenterClient/SecurityCenterClient+Live.swift

```diff
@@ -68,6 +72,11 @@ extension SecurityCenterClient {
 						await (try? deviceFactorSourceClient.isSeedPhraseNeededToRecoverAccounts()) ?? false
 					}
 
+					let problem3 = await problem3()
```

Comment:
I would use the same style as the others, `hasProblem3`, but this one can return an optional tuple. So it will be used like `if let (personas, accounts) = hasProblem3() { ... }`

Reply:
In that case it would be
``` Swift
if let (personas, accounts) = hasProblem3(), (personas.count + accounts.count) > 0 {
}
```

meaning the `hasProblem3()` doesn't actually mean that it HAS the problem.

Reply:
no, as I said I think the tuple should be optional, and return nil when the problem is not present. So the check would be inside hasProblem3